_includes/sidebar.html

```diff
@@ -49,8 +49,37 @@
 
         {% include custom/conditions.html %}
 
-        <ul id="mysidebar" class="nav" style="display: none">
-            <li>
+        <ul id="mysidebar" class="nav {% if page.url == '/' or page.url == '/search.html' %}nav--home{% endif %}" style="display: none">
+            <div class="sidenav-arrow"><img src="images/sidenav-arrow.svg"></div>
```

Comment:
Our CI platform (Teamcity) runs an HTML proofer against the Jekyll output. In addition to checking links, it checks that images have alt texts (for accessibility). It's detecting that the sidenav arrow image is missing one of those, so we need to either add an alt text field:

```
<img src="images/sidenav-arrow.svg" alt="Sidenav arrow">
```

or tell the HTML proofer to ignore the image:

```
<img src="images/sidenav-arrow.svg" data-proofer-ignore>
```